backend/app/core/config.py

```
@@ -56,6 +56,7 @@ class Settings(BaseSettings):
     TEST_MODE: bool = os.environ.get("TEST_MODE", "").lower() == "true"
 
     APP_SYMMETRIC_ENCRYPTION_KEY: str = Field(
+        default_factory=lambda: Fernet.generate_key().decode() if os.environ.get("TESTING", "").lower() == "true" or os.environ.get("DOCS_GENERATION", "").lower() == "true" else "",
```

Pylint (3.3.7)

[convention] Line too long (181/100)

(C0301)